model/role.go

```diff
@@ -115,6 +108,23 @@ func init() {
 		PERMISSION_SYSCONSOLE_WRITE_SITE.Id: {
 			PERMISSION_EDIT_BRAND,
 		},
+		PERMISSION_SYSCONSOLE_WRITE_COMPLIANCE_DATA_RETENTION.Id: {
+			PERMISSION_CREATE_DATA_RETENTION_JOB,
+		},
+		PERMISSION_SYSCONSOLE_READ_COMPLIANCE_DATA_RETENTION.Id: {
+			PERMISSION_READ_JOBS,
+		},
+		PERMISSION_SYSCONSOLE_WRITE_COMPLIANCE_COMPLIANCE_EXPORT.Id: {
+			PERMISSION_CREATE_COMPLIANCE_EXPORT_JOB,
+			PERMISSION_DOWNLOAD_COMPLIANCE_EXPORT_RESULT,
+		},
+		PERMISSION_SYSCONSOLE_READ_COMPLIANCE_COMPLIANCE_EXPORT.Id: {
+			PERMISSION_READ_JOBS,
```

Comment:
And likewise, PERMISSION_READ_COMPLIANCE_EXPORT_JOB?

Reply:
Nice catch @maxerenberg